dynamo-event-to-datacite-request/src/main/java/no/unit/nva/events/models/AwsEventBridgeDetail.java

```diff
@@ -0,0 +1,149 @@
+package no.unit.nva.events.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Objects;
+import nva.commons.utils.JacocoGenerated;
+
+public class AwsEventBridgeDetail<I> {
```

Comment:
I for Input? Call it INPUT? :)

Reply:
I tried and I got this error: "Generics names should be a one letter long and upper case."  https://pmd.github.io/pmd-6.23.0/pmd_rules_java_codestyle.html#genericsnaming

Reply:
Stupid PMD rule IMO :upside_down_face: